src/main/java/io/javalin/core/util/Util.kt

```diff
@@ -16,6 +16,9 @@ object Util {
 
     var noServerHasBeenStarted = true
 
+    fun normalizeContextPath(contextPath: String) = ("/" + contextPath).replace("/{2,}".toRegex(), "/").removeSuffix("/")
+    fun prefixContextPath(path: String, contextPath: String) = if (path == "*") path else (contextPath + "/" + path).replace("/{2,}".toRegex(), "/")
```

Comment:
why are you not returning the contextPath in the if here?

Reply:
So I read the test so I understand it now. So you are basically always treating * as catch everything?